### PR TITLE
Merge Dev to Main for Prod deployment

### PR DESCRIPTION
**Pull request by jaisantosh**

### CODE REVIEW

File: trustwise/models.py [hunk 1]
```diff
@@ -3,22 +3,23 @@
...
 class UploadData(BaseModel):  # Pydantic Model for data uploaded to the Evaluation endpoint
-    user_id: int
+    user_id: str
     experiment_id: str
     query: str
     context: List[Chunk]
     response: str
-    context_aggregated: str  # Context as a single string
+    api_key: Optional[str]
+    # context_aggregated: str  # Context as a single string
```
Comment (mbarker-tw) [inline trustwise/models.py:18]: Can we remove this comment?

File: trustwise/request.py [hunk 2]
```diff
@@ -1,56 +1,36 @@
...
-    else:
-        logging.error(f"API Key is invalid!, Please visit -> {os.getenv('github_login_url')}")
-        raise ValueError("API Key is invalid!")
-
-    # Evaluation URL
-    eval_url = os.getenv('evaluation_url')
+def request_eval(user_id, experiment_id, query, response, api_key=None):
 
-    context_aggregated = ''  # Context retrieved from the RAG pipeline to be stored as a string for evaluations
+    # context_aggregated = ''  # Context retrieved from the RAG pipeline to be stored as a string for evaluations
```
Comment (mbarker-tw) [inline trustwise/request.py:15]: Can we remove this comment?

File: trustwise/request.py [hunk 3]
```diff
@@ -1,56 +1,36 @@
...
     context = []  # Context chunks to be logged in the record with text, score and node id.
 
     for node in response.source_nodes:
-        context_aggregated += node.text
+        # context_aggregated += node.text
```
Comment (mbarker-tw) [inline trustwise/request.py:19]: Same thing, can we remove this comment?

File: trustwise/utils.py [hunk 4]
```diff
@@ -18,9 +24,9 @@ def validate_api_key(api_key):
             user_id = response.json()
             return user_id
         else:
-            logging.error(f"API Key is invalid!, Please visit -> {os.getenv('github_login_url')}")
+            logger.error(f"API Key is invalid!, Please visit -> http://35.199.62.235:8080/github-login")
             return None
```
Comment (mbarker-tw) [inline trustwise/utils.py:28]: Should this be returning None when there's an error? Can't we just delete the "return None" line?

File: trustwise/request.py [hunk 5]
```diff
@@ -1,56 +1,36 @@
...
 
     # Data to be uploaded to the endpoint for the evaluations to run
     data = UploadData(user_id=user_id, experiment_id=experiment_id, query=query, context=context,
-                      response=response.response, context_aggregated=context_aggregated)
+                      response=response.response, api_key=api_key)
 
     try:
         data_dict = data.model_dump()  # Convert to dict for JSON serialization
-        response_object = send_request(url=eval_url, data_dict=data_dict)
+        response_object = requests.post(url=TW_EVALUATION_URL, json=data_dict)
```
Comment (mbarker-tw) [inline trustwise/request.py:33]: Should we add an explicit timeout here? Say 1 min?
```suggestion
        response_object = requests.post(url=TW_EVALUATION_URL, json=data_dict, timeout=60)
```

File: trustwise/callback.py
Comment (nbhatnagar-tw) [inline trustwise/callback.py:1]: Print trace map is used to print the trace in the notebook and does not need logger -  it prints the trace for the user.

Comment (jaisantosh) [reply trustwise/callback.py:1]: just didnt want both logger and print.

Comment (nbhatnagar-tw) [reply trustwise/callback.py:1]: logger isn't needed when printing trace - this is the print trace for the user. so we can remove this change

**Pull request merged**